backend/src/database/data/document_item.py

```diff
@@ -10,6 +10,6 @@ class DocumentItem:
     document_type: str | None = None
     extracted_data: dict[str, Any] | None = None
 
-
-def document_item_to_dict(document_item) -> dict[str, Any]:
-    return {k: v for k, v in asdict(document_item).items() if v is not None}
+    @staticmethod
+    def document_item_to_dict(document) -> dict[str, Any]:
+        return {k: v for k, v in asdict(document).items() if v is not None}
```

Comment:
You don't need this to be a `@staticmethod` because it should be able to work in itself.

```suggestion
    def to_dict(self) -> dict[str, Any]:
        return {k: v for k, v in asdict(self).items() if v is not None}
```

Then you call it by doing something like...
```python
asdf = DocumentItem(fields, to, be, filled, in)
dictionary_version = asdf.to_dict()
```

Reply:
This change seems to work even with the warning from `asdict()`.